CHANGELOG.md

```diff
@@ -54,7 +58,8 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 - (ante) [#1717](https://github.com/evmos/ethermint/pull/1717) Reuse sender recovery result.
 - (cli) [#246](https://github.com/crypto-org-chain/ethermint/pull/246) Call app.Close to cleanup resource on graceful shutdown.
-
+- (precompile) [#380](https://github.com/crypto-org-chain/ethermint/pull/380) Allow init precompiled contract with rules when new evm.
+- (precompile) [#383](https://github.com/crypto-org-chain/ethermint/pull/383) Allow init precompiled contract with ctx.
```

Comment:
Is there any reason why we shouldn't just use the `crypto-org-chain/ethermint` for directly?

Looks like they've already upgrade it to cosmos v0.50.x too

Reply:
It's a good point. We wouldn't be maintaining directly geth and ethermint.

On the tradeoffs side:
- upgrading to cosmos 0.50 is out of the scope for this change and would likely require more work to adapt the codebase.
- depending on cronos ethermint means that we also have to depend on cronos go-ethereum

@skosito @lumtis what are your thoughts on this?

Reply:
crypto.org use their own fork of the Cosmos SDK and made some modification to it. It would be preferrable for us to keep using the official Cosmos SDK